core-addon/IonCore.js

```diff
@@ -0,0 +1,22 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+module.exports = class IonCore {
+  initialize() {
```

Comment:
Any reason not to make this `constructor` so it's not necessary for the caller to call this method too?

Reply:
Only testing convenience! Happy to change this moving forward as required.